src/webhooks/sentry-options/sentry-options.ts

```diff
@@ -80,117 +79,117 @@ export async function messageSlack(message: SentryOptionsResponse) {
   if (message.source !== 'options-automator') {
     return;
   }
-  const successBlock: KnownBlock[] = [
-    slackblocks.header(
-      slackblocks.plaintext(
-        message.region
-          ? `✅ Successfully Updated Options in ${message.region}: ✅`
-          : '✅ Successfully Updated Options ✅'
-      )
-    ),
-    ...(message.updated_options.length > 0
-      ? [
-          slackblocks.divider(),
-          slackblocks.section(slackblocks.markdown('*Updated options:* ')),
-          slackblocks.sectionBlock(
-            message.updated_options.map((option) =>
-              slackblocks.markdown(
-                `updated \`${option.option_name}\` with db value \`${option.db_value}\` to value \`${option.value}\``
+  try {
```

Comment:
I think this try block was in the right place last time, feels like it should be done before sending the slack message, because that's really where a lot of unexpected behavior should occur. It should not happen in this part of the code

Reply:
I put it here because it'll catch a malformed message, and then report that message as a sentry error

Reply:
I still don't think this should be necessary because there should be testing catching the malformed message before it is even sent out. However, if you do think there's a possibility of that happening, guess it is fine to keep here. I'm not super familiar with payloads being sent to this webhook.